disttask/framework/scheduler/interface.go

```diff
@@ -26,12 +26,12 @@ type TaskTable interface {
 	GetGlobalTaskByID(taskID int64) (task *proto.Task, err error)
 
 	GetSubtaskInStates(instanceID string, taskID int64, step int64, states ...interface{}) (*proto.Subtask, error)
-	StartSubtask(id int64) error
-	UpdateSubtaskStateAndError(id int64, state string, err error) error
-	FinishSubtask(id int64, meta []byte) error
```

Comment:
Do we also need to add for these three?

Reply:
`subtaskID` is unique enough to locate the record.